src/common/types/neo4j.ts

```diff
@@ -0,0 +1,18 @@
+export type BasicNode = {
+  id: string
```

Comment:
I notice there's no copy right header on a couple of new files

Reply:
thanks! 👍